prime-router/src/main/kotlin/azure/batch/BatchDeciderFunction.kt

```diff
@@ -46,12 +46,12 @@ class BatchDeciderFunction(private val workflowEngine: WorkflowEngine = Workflow
                     // any that should have batched in the last 60 seconds, get count of outstanding BATCH records
                     //  (how many actions with BATCH for receiver
                     .forEach { rec ->
-                        val (queueMessages, isEmpty) = determineQueueMessageCount(rec, txn)
+                        val (queueMessages, willSendEmptyBatch) = determineQueueMessageCount(rec, txn)
```

Comment:
This boolean signifies that the receiver should send an empty batch, not just that the batch is empty. Open to naming suggestions that are more concise/descriptive.